test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13635,6 +13635,12 @@ def run(self):
             for _l in _sc:
                 (cmd, out) = _l
                 self.logger.info('sending: ' + cmd)
+                # only way I could figure out to catch a sleep command
+                # within a spawned pexpect child. Might need revisiting
+                if 'sleep' in _l[0]:
+                    _secs = _l[0].split()[1]
+                    self.logger.debug('sleeping ' + str(_secs))
+                    time.sleep(float(_secs))
```

Comment:
A change to the PTL framework would need a separate PP ticket.

Reply:
Now i am using script, so there is no need to change in library.